R/DMMParam.R

```diff
@@ -0,0 +1,203 @@
+#' DMM clustering
+#' 
+#' Run the \code{DMM} function on a distance matrix 
+#' within \code{\link{clusterRows}}.
+#' 
+#' @param k An integer indicating the number of clusters, or a list of integers 
+#'   defining the possible number of clusters to choose from.
+#' @param type A string specifying the fit function to use to find the optimal 
+#'   number of clusters. Use in combination with k being a list of integers. 
+#'   Must be equal to \code{laplace}, \code{AIC} or \code{BIC}.
```

Comment:
If these refer to strings rather than functions, you should use `\code{"laplace"}`, etc.